experiments/template_SLTC.py

```diff
@@ -64,6 +66,12 @@ def main():
     if data_args.disable_caching:
         disable_caching()
 
+    if data_args.finetuning_task in ["swiss_criticality_prediction_bge_facts",
+                                     "swiss_criticality_prediction_bge_considerations",
+                                     "swiss_criticality_prediction_citation_facts",
+                                     "swiss_criticality_prediction_citation_considerations"]:
+        os.environ["WANDB_API_KEY"] = "6f9fce3d2b3e41f8880b3e0b094e16ec9d030315"  # wandb rstern token
```

Comment:
there should never be any secrets on GitHub in the open!
This should be put in an .env file for example and loaded from there